doc/template.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-  <title>__SECTION__ Node.js __VERSION__ Manual &amp; Documentation</title>
+  <title>__SECTION__ | Node.js __VERSION__ Manual &amp; Documentation</title>
```

Comment:
Maybe drop the `Manual &amp;` while we're here? Long titles like this usually get truncated on tabs.